modules/ml/src/test/java/org/apache/ignite/ml/LocalModelsTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.ml;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.apache.ignite.ml.clustering.KMeansLocalClusterer;
+import org.apache.ignite.ml.clustering.KMeansModel;
+import org.apache.ignite.ml.math.EuclideanDistance;
+import org.apache.ignite.ml.math.impls.matrix.DenseLocalOnHeapMatrix;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Tests for models import/export functionality.
+ */
+public class LocalModelsTest {
+    /** */
+    private String mdlFilePath = "model.mlmod";
+
+    /**
+     *
+     */
+    @After
+    public void cleanUp() throws IOException {
+        Files.deleteIfExists(Paths.get(mdlFilePath));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void importExportKMeansModelTest(){
```

Comment:
consider adding some code that would cover testing of KMeansModel#distanceMeasure - currently IDEA reports this method as unused

Reply:
this changes(and test too) about import/export, not about kmeans model. But yes we should increase coverage.